### PR TITLE
[chore][site] Add info about using Deckhouse CLI

### PR DESCRIPTION
**Pull request by z9r5**
## Description
Site Ingress configuration minor changes.
Add a page for downloading Deckhouse CLI.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: docs
type: chore
summary: Add a page for downloading Deckhouse CLI.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @sprait. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @elenashliaga. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @elenashliaga. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @elenashliaga. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @elenashliaga. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @elenashliaga. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @elenashliaga. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @elenashliaga. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/pages/DECKHOUSE-CLI_RU.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+Deckhouse CLI — это интерфейс командной строки для работы с кластерами Deckhouse Kubernetes Platform (DKP), который автоматически устанавливается вместе с платформой. Утилиту можно дополнительно [установить](#как-установить-deckhouse-cli) на любую машину.
+
+В командной строке к утилите можно обратиться как `d8`. Она состоит из трех компонентов:
+* `d8 k` — команды, которые в кластерах Kubernetes выполняет `kubectl`.  
+    Например, в кластере DKP можно выполнить `kubectl get pods` как `d8 k get pods`.
+* `d8 d` — команды, отвечающие за доставку по аналогии с утилитой `werf`.  
+    Например, вместо `werf plan --repo registry.deckhouse.io` можно выполнить `d8 d plan --repo registry.deckhouse.io`.
+* `d8 v` — команды, отвечающие за работу с виртуальными машинами.  
+    Например, команда `d8 virtualziation console` подключает к консоли виртуальной машины.
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-CLI_RU.md:15]:
```suggestion
    Например, команда `d8 virtualization console` подключает к консоли виртуальной машины.
```

File: docs/documentation/pages/DECKHOUSE-CLI_RU.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    </ul>
+    </details>
+    </div>
+
+## Как установить Deckhouse CLI?
+
+1. Скачайте архив с подходящей версией ОС:
+    * [Linux x86-64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.3/d8-v0.0.3-linux-amd64.tar.gz)
+    * [macOS x86-64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.3/d8-v0.0.3-darwin-amd64.tar.gz)
+    * [macOS ARM64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.3/d8-v0.0.3-darwin-arm64)
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-CLI_RU.md:34]:
```suggestion
   * [Linux x86-64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.3/d8-v0.0.3-linux-amd64.tar.gz)
   * [macOS x86-64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.3/d8-v0.0.3-darwin-amd64.tar.gz)
   * [macOS ARM64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.3/d8-v0.0.3-darwin-arm64)
```

File: docs/documentation/pages/DECKHOUSE-CLI_RU.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+В командной строке к утилите можно обратиться как `d8`. Она состоит из трех компонентов:
+* `d8 k` — команды, которые в кластерах Kubernetes выполняет `kubectl`.  
+    Например, в кластере DKP можно выполнить `kubectl get pods` как `d8 k get pods`.
+* `d8 d` — команды, отвечающие за доставку по аналогии с утилитой `werf`.  
+    Например, вместо `werf plan --repo registry.deckhouse.io` можно выполнить `d8 d plan --repo registry.deckhouse.io`.
+* `d8 v` — команды, отвечающие за работу с виртуальными машинами.  
+    Например, команда `d8 virtualziation console` подключает к консоли виртуальной машины.
+
+    <div markdown="0">
+    <details><summary>Больше команд для виртуализации</summary>
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-CLI_RU.md:18]:
```suggestion
    <details><summary>Больше команд для виртуализации...</summary>
```

Comment (elenashliaga) [reply docs/documentation/pages/DECKHOUSE-CLI_RU.md:18]: зачем?

Comment (z9r5) [reply docs/documentation/pages/DECKHOUSE-CLI_RU.md:20]: Предложение предполагает что это раскрывашка. По аналоги как у внешней ссылки ставят стрелочку, что это внешняя ссылка.
Как другой пример - _show more..._   и такого плана.
У нас в доке так во многих местах. Если хотите по другому -> в глоссарий и править всю доку. Я просто обратил внимание.

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @elenashliaga. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @elenashliaga. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/pages/DECKHOUSE-CLI_RU.md (hunk 3 above)
Comment (z9r5) [reply docs/documentation/pages/DECKHOUSE-CLI_RU.md:20]: Я бы не считал это заголовком. Тогда надо ставить знак препинания в конце. Либо точку, либо три точки как я выше предлагал.

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @elenashliaga. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @elenashliaga. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/pages/DECKHOUSE-CLI_RU.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    </div>
+
+## Как установить Deckhouse CLI?
+
+1. Скачайте архив с подходящей версией ОС:
+   * [Linux x86-64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.3/d8-v0.0.3-linux-amd64.tar.gz)
+   * [macOS x86-64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.3/d8-v0.0.3-darwin-amd64.tar.gz)
+   * [macOS ARM64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.3/d8-v0.0.3-darwin-arm64)
+
+1. Распакуйте архив командой:
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-CLI_RU.md:36]:
```suggestion
1. Распакуйте архив:
```
Выбивается из стилистики остальных пунктов

File: docs/documentation/pages/DECKHOUSE-CLI_RU.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    Например, команда `d8 virtualization console` подключает к консоли виртуальной машины.
+
+    <div markdown="0">
+    <details><summary>Больше команд для виртуализации...</summary>
+    <ul>
+    <li><code>d8 v console</code> подключает к консоли виртуальной машины.</li>
+    <li><code>d8 v port-forward</code> перенаправляет локальные порты на виртуальную машину.</li>
+    <li><code>d8 v scp</code> использует клиент SCP для работы с файлами на виртуальной машине.</li>
+    <li><code>d8 v ssh</code> настроит SSH-соединение с виртуальной машиной.</li>
+    <li><code>d8 v vnc</code> настроит VNC-соединение с виртуальной машиной.</li>
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-CLI_RU.md:24]:
```suggestion
    <li><code>d8 v console</code> — подключает к консоли виртуальной машины.</li>
    <li><code>d8 v port-forward</code> — перенаправляет локальные порты на виртуальную машину.</li>
    <li><code>d8 v scp</code> — использует клиент SCP для работы с файлами на виртуальной машине.</li>
    <li><code>d8 v ssh</code> — устанавливает SSH-соединение с виртуальной машиной.</li>
    <li><code>d8 v vnc</code> — устанавливает VNC-соединение с виртуальной машиной.</li>
```

File: docs/documentation/pages/DECKHOUSE-CLI_RU.md (hunk 4 above)
Comment (elenashliaga) [reply docs/documentation/pages/DECKHOUSE-CLI_RU.md:36]: ок

File: docs/documentation/pages/DECKHOUSE-CLI_RU.md (hunk 5 above)
Comment (z9r5) [reply docs/documentation/pages/DECKHOUSE-CLI_RU.md:24]: ну можно  - подключает по SSH к виртуальной машине

Comment (z9r5) [reply docs/documentation/pages/DECKHOUSE-CLI_RU.md:24]: А в первом пункте исправил, т.к. три пункта про подключение - написано не в одном стиле.

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/pages/DECKHOUSE-CLI_RU.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+Deckhouse CLI — это интерфейс командной строки для работы с кластерами от Deckhouse Kubernetes Platform (DKP). Начиная с релиза 1.59, интерфейс автоматически устанавливается на все узлы кластера. Утилиту можно также [установить](#как-установить-deckhouse-cli) на любую машину и использовать для работы с кластерами без DKP.
+
+В командной строке к утилите можно обратиться как `d8`. Все команды сгруппированы по функциям:
+* `d8 k` — команды, которые в кластерах Kubernetes выполняет `kubectl`.  
+    Например, в кластере можно выполнить `kubectl get pods` как `d8 k get pods`.
+* `d8 d` — команды, отвечающие за доставку по аналогии с утилитой `werf`.  
+    Например, вместо `werf plan --repo registry.deckhouse.io` можно выполнить `d8 d plan --repo registry.deckhouse.io`.
+* `d8 v` — команды, отвечающие за работу с виртуальными машинами, созданными [Deckhouse Virtualization Platform](../../modules/virtualization/stable/).
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-CLI_RU.md:14]:
```suggestion
* `d8 v` — команды, отвечающие за работу с виртуальными машинами, созданными [Deckhouse Virtualization Platform](../../../../modules/virtualization/stable/).  
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/pages/DECKHOUSE-CLI_RU.md (hunk 5 above)
Comment (elenashliaga) [reply docs/documentation/pages/DECKHOUSE-CLI_RU.md]: обсудили в Салки

File: docs/documentation/pages/DECKHOUSE-CLI_RU.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+Deckhouse CLI — это интерфейс командной строки для работы с кластерами от Deckhouse Kubernetes Platform (DKP). Начиная с релиза 1.59, интерфейс автоматически устанавливается на все узлы кластера. Утилиту можно также [установить](#как-установить-deckhouse-cli) на любую машину и использовать для работы с кластерами без DKP.
+
+В командной строке к утилите можно обратиться как `d8`. Все команды сгруппированы по функциям:
+* `d8 k` — команды, которые в кластерах Kubernetes выполняет `kubectl`.  
+    Например, в кластере можно выполнить `kubectl get pods` как `d8 k get pods`.
+* `d8 d` — команды, отвечающие за доставку по аналогии с утилитой `werf`.  
+    Например, вместо `werf plan --repo registry.deckhouse.io` можно выполнить `d8 d plan --repo registry.deckhouse.io`.
+* `d8 v` — команды, отвечающие за работу с виртуальными машинами, созданными [Deckhouse Virtualization Platform](../../../../modules/virtualization/stable/).
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-CLI_RU.md:14]:
```suggestion
* `d8 v` — команды, отвечающие за работу с виртуальными машинами, созданными [Deckhouse Virtualization Platform](/modules/virtualization/stable/).  
```

**Review by MariaMakeeva — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

File: docs/documentation/pages/DECKHOUSE-CLI.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+    </ul>
+    </details>
+    </div>
+
+## How do I install Deckhouse CLI?
+
+1. Download the archive for your OS/architecture:
+   * [Linux x86-64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.3/d8-v0.0.3-linux-amd64.tar.gz)
+   * [macOS x86-64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.3/d8-v0.0.3-darwin-amd64.tar.gz)
+   * [macOS ARM64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.3/d8-v0.0.3-darwin-arm64)
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-CLI.md:33]:
```suggestion
   * [Linux x86-64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.5/d8-v0.0.5-linux-amd64.tar.gz)
   * [macOS x86-64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.5/d8-v0.0.5-darwin-amd64.tar.gz)
   * [macOS ARM64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.5/d8-v0.0.5-darwin-arm64)
```

File: docs/documentation/pages/DECKHOUSE-CLI_RU.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+    </ul>
+    </details>
+    </div>
+
+## Как установить Deckhouse CLI?
+
+1. Скачайте архив с подходящей версией ОС:
+   * [Linux x86-64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.3/d8-v0.0.3-linux-amd64.tar.gz)
+   * [macOS x86-64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.3/d8-v0.0.3-darwin-amd64.tar.gz)
+   * [macOS ARM64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.3/d8-v0.0.3-darwin-arm64)
```
Comment (z9r5) [inline docs/documentation/pages/DECKHOUSE-CLI_RU.md:34]:
```suggestion
   * [Linux x86-64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.5/d8-v0.0.5-linux-amd64.tar.gz)
   * [macOS x86-64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.5/d8-v0.0.5-darwin-amd64.tar.gz)
   * [macOS ARM64]({% if site.mode == 'module' %}{{ site.urls[page.lang] }}{% endif %}/downloads/deckhouse-cli/v0.0.5/d8-v0.0.5-darwin-arm64)
```

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [8077](https://github.com/deckhouse/deckhouse/pull/8077) to the branch [release-1.59](https://github.com/deckhouse/deckhouse/tree/release-1.59) successful!

**Pull request merged**